sphinx/ext/autosummary/generate.py

```diff
@@ -216,11 +217,25 @@ def scan(self, imported_members: bool) -> List[str]:
                 # list all members up
                 members.append(name)
             elif imported is False:
-                # list not-imported members up
+                # list not-imported members
+                members.append(name)
+            elif '__all__' in dir(self.object) and not self.app.config.autosummary_ignore_module_all:
+                # list members that have __all__ set
                 members.append(name)
 
         return members
 
+def members_of(conf: Config, obj: Any) -> Sequence[str]:
```

Comment:
The main target of this function is `obj`. So it would be better to move it to the first argument of the arglist. Sorry for the noise. But I think the devil's in the details.

Reply:
No worries :)